.github/workflows/go_lint_and_test.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           go-version: ${{ inputs.go_version }}
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@v8
         with:
           version: v1.64.5
```

Comment:
We need to confirm that `golangci/golangci-lint-action@v8` supports `golangci-lint` `v1.64.5` or upgrade both versions.